docs/readme.ptxt

```diff
@@ -36,85 +36,97 @@ OpenMSX is an open source Music Set for OpenTTD.
 with the windows "dmusic" sound driver. You may want to try the "win32" music
-driver, if you have problems with windows' default dmusic driver. Search your
+driver, if you have problems with Windows' default dmusic driver. Search your
 openttd.cfg for "musicdriver" and change the line to "musidriver = win32".
 Consult also the OpenTTD readme for available options and its known-bugs.txt
-for a more detailed and possibly more up to date description of windows music
+for a more detailed and possibly more up to date description of Windows music
 driver issues.
 
+(GNU/Linux)
+If you cannot get music to play, it might be because your system does not
+support MIDI playback yet.
+Please refer to the manual of your Linux distribution to learn about how
+to get MIDI files to play. We recommend to install TiMidity++.
 
 
 3 Contributing and compiling from source
 ========================================
 
-The OpenMSX source is available in a Mercurial repository or as gzip'ed tarball.
-You can do an anonymous checkout from http://mz.openttdcoop.org/hg/openmsx ,
+The OpenMSX source is available in a Git repository or as gzip'ed tarball.
+You can do a clone from
+   https://github.com/OpenTTD/OpenMSX
 e.g. using
-   hg clone http://mz.openttdcoop.org/hg/openmsx or obtain the tarball from
-http://bundles.openttdcoop.org/openmsx/releases.
+   git clone https://github.com/OpenTTD/OpenMSX
+or obtain the tarball from
+   https://www.openttd.org/downloads/openmsx-releases/latest.html
 
 Prerequisites to building OpenMSX:
-- mercurial (only when not building from a tarball, available from
-http://mercurial.selenic.com/wiki/Download?action=show&redirect=BinaryPackages)
-- md5sum (linux, mingw) or md5 (mac)
-- some gnu utils: make, cat, sed
+- Git (only when not building from a tarball)
+- md5sum (Linux, MinGW) or md5 (macOS)
+- some GNU utils: make, cat, sed
 - unix2dos for convenient conversion of the text files
-- python (if you use mercurial, you'll have python)
+- Python
 and you might additionally want a text editor of your choice and possibly a
 programme capable of creating and editing midi files.
 
-On Windows: we advise to get a mingw development environment, download mercurial
-from the sources mentioned above) For more detailed instructions see our guide
+On Windows: We advise to get a MinGW development environment.
+For more detailed instructions, see our guide
 at http://dev.openttdcoop.org/projects/home/wiki and the very extensive and
-detailed installation instructions on the mingw wiki at
+detailed installation instructions on the MinGW wiki at
 http://www.mingw.org/wiki/Getting_Started
 
-On Linux: your system should already have most tools, you'll probably only
-mercurial available from the source mentioned above. For installation
-instructions concerning mercurial refer to the manual of your distribution.
+On GNU/Linux: Your system should already have most tools, you'll probably only
```

Comment:
"you'll probably only Git available" - doesn't make sense (but the original was like this too). There's now no source mentioned above, and Git is included with the majority of distributions, so perhaps just get rid of the ", you'll probably only .... above." bit.